Formula/s/simdutf.rb

```diff
@@ -1,11 +1,21 @@
 class Simdutf < Formula
   desc "Unicode conversion routines, fast"
   homepage "https://simdutf.github.io/simdutf/"
-  url "https://github.com/simdutf/simdutf/archive/refs/tags/v7.3.4.tar.gz"
-  sha256 "c42ed66ceff7bc3e5f4981453864d1b7f656032843909b3807a632be46a1f5d4"
   license any_of: ["Apache-2.0", "MIT"]
   head "https://github.com/simdutf/simdutf.git", branch: "master"
 
+  # Remove `stable` block when patch is no longer needed.
+  stable do
+    url "https://github.com/simdutf/simdutf/archive/refs/tags/v7.3.5.tar.gz"
+    sha256 "3fbc43a4edda3858b89648b7fcaedc19187a9fd22234bed8bb644bbadcf83a79"
+
+    # Fix missing `benchmarks/find`. Remove in next release.
+    patch do
+      url "https://github.com/simdutf/simdutf/commit/46d166bcda0dbab2fd5a008e2f976a3804327550.patch?full_index=1"
+      sha256 "c14b945320500d84a0f6b972c96606f0eaf9db026517c74f277086efebacb450"
+    end
```

Comment:
Might get retagged given this, though, especially since this tag hasn't been published as a GitHub release.

Reply:
Yep, that's exactly what happened